docs/lib/storage/fragments/js/download.md

```diff
@@ -12,25 +12,78 @@ await Storage.get(key: string, config: {
 })
 ```
 
-`Storage.get` returns a signed URL `string` to your file if `download` is false, which is the default.
+`Storage.get` returns a signed URL `string` to your file, if `download` is false, which is the default. You can use this to create a download link for people to click on. This is our recommended option. Note that this method **does not check if the file actually exists** as that would involve an extra API call.
 
-`Storage.get` returns a [`Blob`](https://developer.mozilla.org/en-US/docs/Web/API/Blob) of object data if `download` is true.
+```js
+// get the signed URL string
+const signedURL = await Storage.get(file.__data.key)
 
-### File download
+// inside your template or JSX code. Note <a download> doesn't work here because it is not same origin
+<a href={signedURL} target="_blank">{fileName}</a>
+```
+
+If `download` is true, `Storage.get` returns an object with a `Body` field of type [`Blob`](https://developer.mozilla.org/en-US/docs/Web/API/Blob). It is up to you to programmatically save it to disk (we suggest a method below) or do whatever else you need with it.
 
-Send object data for immediate file download:
+### File download option
+
+The `download` option send you the object data for download or programmatic manipulation:
 
 ```javascript
 const data = await Storage.get(`filename.txt`, { download: true })
 
-// data is a Blob
+// data.Body is a Blob
```

Comment:
this was a mistake by me in my first PR to this doc, which led to https://github.com/aws-amplify/docs/issues/2512